Shared/qml/About.qml

```diff
@@ -105,7 +105,7 @@ Item {
 
             Text {
                 anchors.horizontalCenter: parent.horizontalCenter
-                text: "Built with ArcGIS Maps SDK for Qt"
+                text: "Built with ArcGIS Maps SDK 200.5 for Qt"
```

Comment:
How can we keep track of this when we change the version?

Reply:
Pushed another commit that will ultimately derive the values from the version number listed in the .pro files.